docs/api-reference/regularization-l1-l2.md

```diff
@@ -0,0 +1,27 @@
+This class uses [empirical risk minimization](https://en.wikipedia.org/wiki/Empirical_risk_minimization)
```

Comment:
It's not clear how this connects with l1-norm and l2-norm regularization. Let me know if you want to discuss this offline.

Reply:
ERM ---> without enough data ---> overfit ---> use regularization to overcome overfit.

---
In reply to: [279009294](https://github.com/dotnet/machinelearning/pull/3586#discussion_r279009294) [](ancestors = 279009294)